app/services/partnerships/report.rb

```diff
@@ -2,7 +2,8 @@
 
 module Partnerships
   class Report < BaseService
-    CHALLENGE_WINDOW = 14.days.freeze
+    INITIAL_CHALLENGE_WINDOW = (2.months - 1.day).freeze
```

Comment:
Can we use better naming here if you want to do this, suggest something like:
DEFAULT_CHALLENGE_WINDOW (2 weeks from now)
EARLY_YEAR_CHALLENGE_WINDOW (the October one)?